fhirzeug/generators/python_pydantic/static_files/tests/test_extension.py

```diff
@@ -0,0 +1,56 @@
+"""Test Extension element."""
+import pytest
+import pydantic
+
+from pydantic_fhir import r4
+
+
+def test_extension_validation() -> None:
```

Comment:
To make it more readable, I'd split this test into smaller ones. e.g. `test_single_subextension`, `test_value_and_subextensions_not_allowed`. Or alternatively, add some comments before checks.